packages/backend/src/nest/common/timed-queue.ts

```diff
@@ -56,7 +56,8 @@ export class TimedQueue {
       throw new Error(`Backoff factor must be a positive number greater than or equal to 1`)
     }
 
-    this.queue = fastq.promise(this._processQueue.bind(this), options.concurrency ?? DEFAULT_CONCURRENCY)
+    this._processQueue = this._processQueue.bind(this)
```

Comment:
All of the places where I updated the bind to be on a separate line are because I found while working on the QSS branch that doing it inline with the assignment as a function parameter didn't work properly but doing it like this did.